gneiss/_summary.py

```diff
@@ -57,3 +58,72 @@ def __init__(self, stat_results,
         # calculate the overall coefficient of determination (i.e. R2)
         sst = sse + ssr
         self.r2 = 1 - sse / sst
+
+    def _check_projection(self, project):
```

Comment:
OK, @mortonjt and I discussed this offline and it makes sense to keep as is. Basically, this is converting the results to one of two formats based on the projection.